flake.nix

```diff
@@ -0,0 +1,40 @@
+{
+  description = "Dev shell for Pulumi to configure network services";
+
+  inputs = {
+    nixpkgs.url = "nixpkgs/nixos-unstable";
+    flake-parts.url = "github:hercules-ci/flake-parts";
+  };
+
+  outputs = { ... }@inputs:
+    inputs.flake-parts.lib.mkFlake { inherit inputs; } {
+      systems = [ "aarch64-darwin" "x86_64-linux" ];
+
+      perSystem = { pkgs, ... }:
+        let
+          # Bundle the Pulumi binary with the language package to remove the
+          # warning about them not being in the same directory.
+          # See: https://github.com/pulumi/pulumi/issues/14525
+          pulumiBundle = pkgs.stdenv.mkDerivation {
+            name = "pulumi-bundle-${pkgs.pulumi.version}";
+            phases = [ "installPhase" "fixupPhase" ];
```

Comment:
I'm supposing this is specifying the phases to be defined for this derivation. Unfortunately I was unable to look it up in the stdenv documentation, because there seem to be none. `fixupPhase` isn't defined, so is it needed here?

Reply:
Trying to remember why I originally did this. I think it's to do with skipping the unpack phase because there's no "source code" for this derivation. I'm pretty sure this derivation would error if you ran it without overriding the stages. Yes, I'm pretty sure this isn't clearly documented and I found a solution to my problem when I got errors about not providing a `src` for the derivation

Reply:
This is another thing that has come via `hc-github-config` and `nomad-server`